### ISSUE TITLE
reference manual

### ISSUE THREAD

**Issue by rds99** [facebook/prophet#619]
Hi
Is there a complete reference manual for python?
Thanks

**Comment by bletham** [comment]
There is not. I can point you to the R reference manual (https://cran.r-project.org/web/packages/prophet/prophet.pdf). All of these functions are described at some point in the documentation so you could see how it is called (if it is a method on Prophet or a separate function), and then the inputs are the same for both versions so the descriptions in the R manual would be correct for the Python version also.

But it would be good to auto-convert the Py docstrings into a reference manual the way that CRAN does.

**Comment by ispmarin** [comment]
:+1:

**Comment by irumata** [comment]
This is the still actual problem

**Comment by DEEKSHACodeaim** [comment]
Hi, is this issue open under hacktoberfest?

**Comment by tcuongd** [comment]
Yes, sure if anyone wants to have a go at creating API docs that'd be great. I'm not very experienced in this space but I'm guessing as a minimum we'd want a local version running, then we can think about how to deploy it.
